Livecheckables/mupdf-tools.rb

```diff
@@ -1,4 +1,4 @@
 class MupdfTools
-  livecheck :url   => "https://mupdf.com/downloads/",
+  livecheck :url   => "https://mupdf.com/downloads/archive/?C=M&O=D",
             :regex => /href="mupdf-([0-9\.]+)-source/
```

Comment:
```suggestion
            :regex => /href="mupdf-(\d+(?:\.\d+)+)-source/
```

Could you replace the loose version-matching regex here with the more explicit version that we normally use (as above)? I try to replace these as I find them when editing older livecheckables.

Reply:
Sounds good